### DISCUSSION TITLE
Mouse Movement Continuously Prints Characters in Terminal

### DISCUSSION THREAD

**Discussion by soulrx** [Textualize/textual#29] [Q&A] [answered]
This looks Amazing! I can't wait to use it, but I ran into this problem after 'ctrl-c' or 'q'. After exiting any movement of mouse inserts characters into terminal. All that can be done at that point is to close the terminal. I'm not sure if this is a bug or something wrong in my environment.

**To Reproduce**

I just ran the grid_auto example - python examples/grid_auto.py

I have a screenshot, but don't see how to insert it.

**Platform**

Python 3.9.4 (default, Jul 13 2021, 11:18:03) [GCC 10.3.1 20210422 (Red Hat 10.3.1-1)] on linux (Fedora release 33)
tested on these terminals with same behavior:

  * guake-3.6.3-6.fc33.noarch
  * mate-terminal-1.24.1-1.fc33.x86_64

**Diagnose**

$ pip freeze
```
colorama==0.4.4
commonmark==0.9.1
pyfiglet==0.8.post1
Pygments==2.9.0
rich==10.6.0
textual==0.1.6
```

**python -m rich.diagnose**
```
console width=272 ColorSystem.TRUECOLOR
     color_system = 'truecolor'
         encoding = 'utf-8'
             file = <_io.TextIOWrapper name='<stdout>' mode='w' encoding='utf-8'>
           height = 65
    is_alt_screen = False	
 is_dumb_terminal = False
   is_interactive = True
       is_jupyter = False
      is_terminal = True
   legacy_windows = False
         no_color = False
          options = ConsoleOptions(size=ConsoleDimensions(width=272, height=65),
		    legacy_windows=False, min_width=1, max_width=272, is_terminal=True,
		    encoding=utf-8, max_height=65, justify=None, overflow=None,
		    no_wrap=False, highlight=None, markup=None, height=None)
            quiet = False
           record = False
         safe_box = True
             size = ConsoleDimensions(width=272, height=65)
        soft_wrap = False
           stderr = False
            style = None
         tab_size = 8
            width = 272
```

**python -m rich._windows**
```
platform="Linux"
WindowsConsoleFeatures(vt=False, truecolor=False)
```

Anyone have any ideas what could be going wrong here?

**Comment by willmcgugan** [comment, marked as answer]
Yeah, I've seen this happen on Alacritty terminal, but not any others. Most likely a bug in the code to enable / disable mouse support, but I've haven't figured it out yet.

**Reply by soulrx** [reply]
Thanks for the reply.

Super early yeah.  

Heard about this project on "Talk Python to Me". I had to keep going back 10 seconds to hear "textual".

Looks great.  Keep up the great work.
